torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -1005,6 +1005,52 @@ TensorView* sum_to(TensorView* in, const std::vector<Int*>& sum_to_size) {
   return out;
 }
 
+TensorView* sum_to(TensorView* in, const std::vector<int64_t>& sum_to_size) {
+  const auto& root = TensorDomain::noReductions(in->getRootDomain());
+
+  TORCH_CHECK(
+      root.size() >= sum_to_size.size(),
+      "sum_to: Error trying to reduce",
+      in,
+      "into a shape of size",
+      sum_to_size.size());
+
+  // If no reduction is needed sum_to returns the input tv
+  TensorView* out = in;
+
+  const int64_t leading_dims = root.size() - sum_to_size.size();
+
+  // Generate reduction axes for leading dims
+  std::vector<int> reduce_dims(leading_dims);
```

Comment:
std::vector<int> is needed because of our signature here: https://github.com/csarofeen/pytorch/blob/9038bb4d66f71fd1eaf31c31b4d12741df273560/torch/csrc/jit/codegen/cuda/arith.h#L123

I thinks `int` is pretty safe, given that we are dealing dimensions here, instead of size. So unlikely that we are dealing anything larger than 8.